honda_odyssey_exl_2018.dbc

```diff
@@ -111,6 +114,11 @@ BO_ 432 STANDSTILL: 7 VSA
  SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] "" EON
 
 BO_ 450 XXX_6: 8 XXX
+ SG_ EPB_Brake_and_Pull_Indicator : 6|1@0+ (1,0) [0|1] "" XXX
```

Comment:
@joeljacobs can you use CAPITAL notation here too? other than that it looks good